src/stateManager/StateManager.ts

```diff
@@ -1551,6 +1555,11 @@ class StateManager {
         }
 
         // step 11 - schedule a timeout check for the next participant
+        // Don't schedule timeout tasks if StateManager is disposed
+        if (this.isDisposed) {
+            return;
```

Comment:
Imo not needed, since `timeoutManager.scheduleTask` considers disposal when scheduling